lib/takeaway_menu.rb

```diff
@@ -0,0 +1,58 @@
+require_relative 'menu'
+require_relative 'text'
+
+class Takeaway
+  attr_accessor :total
+  attr_reader :dishes
+
+  def initialize
+    @dishes = Menu.new.dishes
+    @basket = {}
+    @total = 0
+    @send_text = Text.new
+  end
+
+  def order(dish, quantity = 1)
```

Comment:
clever way of getting the quantity in.. I missed this entirely and it caused carnage